docs/monitoring.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Monitoring
+description: Learn how to ensure your actors, tasks and storages are running as you intended. Validate your data and receive notifications when your jobs succeed or fail.
+menuWeight: 5
+category: guides
+paths:
+    - monitoring
+---
+
+# [](#what-is-the-monitoring-suite) What is the monitoring suite?
+
+The [monitoring suite](https://apify.com/apify/monitoring) is a collection of Apify [actors]({{@link actors.md}}) that allows you to automate the monitoring of jobs you have running on the Apify [platform](https://apify.com).
+
+You can use it to monitor anything from a single actor to complex projects spanning multiple actors, [tasks]({{@link actors/tasks.md}}) or [storages]({{@link storage.md}}).
+
+## [](#how-does-it-work) How does it work?
+
+The monitoring suite is made up of a system of actors we call **checkers**. Each checker is programmed to monitor one thing, e.g. the items in a [dataset]({{@link storage/dataset.md}}) for duplicates and correct format.
+
+Based on your configuration, the suite will create [schedules]({{@link schedules.md}}) and [webhooks]({{@link webhooks.md}}) under your account and use those to run the checkers. Some checkers will create visualizations of how your data progress over time, others will send [notifications](#notofications) about failed runs.
```

Comment:
We have checkers and reporters. The checkers do the testing/data collecting part, but the once responsible of sending notifications and creating the visualization are reporters.